go-controller/pkg/util/ovs.go

```diff
@@ -125,6 +125,67 @@ type execHelper struct {
 
 var runner *execHelper
 
+type ExecUtilRunSvc interface {
```

Comment:
would a more succinct name be better? Liike ExecRunner or something.

Reply:
I am open for suggestions to make the naming better

Reply:
I like ExecRunner

Reply:
Also +1 on `ExecRunner`

Reply:
Appreciate the suggestions

Reply:
As per review suggestion, the latest PR renames it as `ExecRunner`